meta/data.go

```diff
@@ -396,6 +399,8 @@ func (data *Data) CreateShardGroup(database, policy string, timestamp time.Time)
 	for i := range sgi.Shards {
 		data.MaxShardID++
 		sgi.Shards[i] = ShardInfo{ID: data.MaxShardID}
+		data.Logger.Printf("new shard %d successfully created for shard group '%d' database '%s', retention policy '%s'",
```

Comment:
We actually prefer if log messages are coded at the highest possible level, assuming detail is still not lost. Since this function is only called from one place in the code, shouldn't that be the place to add the log message?

Make sense?